backend/corpora/dataset_processing/h5ad_data_file.py

```diff
@@ -187,41 +174,30 @@ def validate_anndata(self):
 
     def extract_anndata_elements_from_file(self):
         logging.info(f"Reading in AnnData dataset: {path.basename(self.input_filename)}")
-        self.anndata = anndata.read_h5ad(self.input_filename, backed="r" if self.backed else None)
+        self.anndata = anndata.read_h5ad(self.input_filename)
         logging.info("Completed reading in AnnData dataset!")
 
         self.obs = self.transform_dataframe_index_into_column(self.anndata.obs, "obs", self.obs_index_column_name)
-        self.var = self.transform_dataframe_index_into_column(self.anndata.var, "var", self.vars_index_column_name)
+        self.var = self.transform_dataframe_index_into_column(self.anndata.var, "var", self.var_index_column_name)
 
     def extract_metadata_about_dataset(self):
         """
         Extract metadata information about the dataset that upon conversion will be saved as group metadata with the
         CXG that is generated. This metadata information includes Corpora schema properties, the dataset title and
         a link that details more information about the dataset.
         """
+        self.corpora_properties = self.get_corpora_properties()
 
-        self.corpora_properties = self.get_corpora_properties() if self.use_corpora_schema else None
-
-        if self.corpora_properties is None and self.use_corpora_schema:
+        if self.corpora_properties is None:
             # If the return value is None, this means that we were not able to figure out what version of the Corpora
             # schema the object is using and therefore cannot extract any properties.
             raise ValueError("Unknown source file schema version is unsupported.")
 
         # The title and about properties of the dataset are set by the following order: if they are explicitly defined
-        # then use the explicit value. If the dataset is a Corpora-schema based schema, then extract the title and about
```

Comment:
we don't have project_links anymore, right?

Reply:
Correct. Even if we were to add project links to the dataset, we probably shouldn't surface them through the CXG which is immutable.